content/user/ui/settings/Roles.md

```diff
@@ -5,32 +5,34 @@ tags: "TeamServer roles observer superadmin system"
 -->
 
 ## System Roles
-System roles are offered for EOP (Enterprise on Premise) customers responsible for administrating and managing the TeamServer. There are three system roles presently:
+Contrast offers system roles for Enterprise on Premise (EOP) customers responsible for administrating and managing the Contrast UI. There are three system roles:
```

Comment:
Talked to Whalen about this already but moving forward, let's consistently say Enterprise on Premises. ("Premises" vs. "Premise" because they mean two different things).